pkg/plugin/plugin.go

```diff
@@ -78,6 +80,11 @@ type EnvArgs struct {
 	OvnPort types.UnmarshallableString `json:"ovnPort,omitempty"`
 }
 
+type cachedNetConf struct {
+	Netconf    *netConf
+	OrigIfName string
```

Comment:
Would you please add a docstring describing what is this good for? I had troubles figuring it out. What is this set to when HWOF is not used?

Reply:
yes, done.